scavenger-agent-java/src/main/java/com/navercorp/scavenger/javaagent/collecting/InvocationTracker.java

```diff
@@ -46,7 +46,7 @@ public static void installAdvice(Instrumentation inst, Config config) {
         Advice advice = Advice.to(InvocationTracker.class);
         AgentBuilder transform = new AgentBuilder.Default(new ByteBuddy().with(TypeValidation.DISABLED))
             .type(matcherBuilder.buildClassMatcher())
-            .transform((builder, typeDescription, classLoader, module) ->
+            .transform((builder, typeDescription, classLoader, module, protectionDomain) ->
```

Comment:
what is protectionDomain for?

Reply:
It's a `java.security.ProtectionDomain` class and when I debug it... `ProtectionDomain` has information about the source location of the code where the class is loaded.